aspnet/mvc/overview/getting-started/introduction/adding-search.md

```diff
@@ -11,14 +11,24 @@ msc.type: authoredcontent
 ---
 Search
 ====================
-by [Rick Anderson]((https://twitter.com/RickAndMSFT))
 
 [!INCLUDE [Tutorial Note](sample/code-location.md)]
 
 ## Adding a Search Method and Search View
 
 In this section you'll add search capability to the `Index` action method that lets you search movies by genre or name.
 
+## Prerequisites
+
+This section requires that you first run the application (F5) and add the following movies to the database.
```

Comment:
Why is this required now but it wasn't in the original version?  Or is this just based on what's in screen shots?  I'm wondering if this is really necessary to follow the tutorial.

Reply:
Hi Tom: I was thinking I would remove the [WIP] when peer review was done and it was time for the technical review. Is that not the process? I added the 'prerequisites' section because, in going thru the tutorial from the beginning, I did not come across any section where those records were added. Only one movie had been added, and not the three shown in screenshots throughout the search article.

Reply:
On second thought, let me check that again. I think I know what might have happened.

Reply:
Yeah, I checked this again and I'm not seeing where these movies get added. I don't see a database switch happening either. One thing I noticed, perhaps unrelated, is that I didn't see a "DefaultConnection" string in my web.config file. Another thing, again, perhaps unrelated, is there was a mismatch between text and screenshot when creating a controller in "Accessing Your Model's Data from a Computer". There is a mismatch with "Data context class" (see attached)
![textimagemismatch](https://user-images.githubusercontent.com/45210983/51275520-0ebc1f80-1987-11e9-8d8a-ba72b217513f.png)

Reply:
I suspect the way the screenshots didn't reflect the data a person would see in their own browser is just how the tutorial was designed.  If you keep the prereqs you might qualify it by saying something like "if you want your screens to match the sample screenshots, enter this data"

On the context name, where is the context created?  Does it say to create it with the ApplicationDbContext name?

If not DefaultConnection, what is the connection string named?

Reply:
Ok on number 1, will do. 
The text reflects the tutorial's instructions up to that point. You can see that it says to 
"Select MovieDBContext (MvcMovie.Models) for the Data context class." 
I didn't see any connection string, but I added the DefaultConnection they said was there, along with the new connection string it said to add.

Reply:
What's confusing things is this PR seems to be a combination of tutorials from at least two different tutorial series -- MVC intro and EF database first.  Would it be difficult to split it up into one PR for each series?